src/app/legacy/containers/StoryPromo/LinkContents/index.jsx

```diff
@@ -78,11 +78,12 @@ const LinkContents = ({ item, isInline, id }) => {
 LinkContents.propTypes = {
   item: shape(pick(['cpsType', 'headlines', 'media'], storyItem)).isRequired,
   isInline: bool,
-  id: string.isRequired,
+  id: string,
```

Comment:
Hmm we might have to live with this one 🤔 - id is set to null if the link is not a live link.. 

https://github.com/bbc/simorgh/blob/latest/src/app/legacy/containers/StoryPromo/index.jsx#L195